docs/source/pythonapi/examples/mgxs-part-iii.ipynb

```diff
@@ -1459,8 +1517,8 @@
      "output_type": "stream",
      "text": [
       "openmc keff = 1.025390\n",
-      "openmoc keff = 1.026401\n",
-      "bias [pcm]: 101.1\n"
+      "openmoc keff = 1.026386\n",
```

Comment:
As discussed above this is because the MGXS didnt change but the printed values did (i.e., rel err was printed vice mean).  

- [x]